src/Microsoft.ML.Legacy/Runtime/EntryPoints/ModelOperations.cs

```diff
@@ -123,5 +125,38 @@ public static ApplyTransformModelOutput Apply(IHostEnvironment env, ApplyTransfo
         {
             return new ApplyTransformModelOutput() { OutputData = input.TransformModel.Apply(env, input.Data) };
         }
+
+        [TlcModule.EntryPoint(Name = "Models.OvaModelCombiner", Desc = "Combines a sequence of PredictorModels into a single model")]
+        public static PredictorModelOutput CombineOvaModels(IHostEnvironment env, CombineOvaPredictorModelsInput input)
```

Comment:
Just for my knowledge - what is the plan for these entry points that we plan on keeping? Do we remove all the "real legacy" stuff from ML.Legacy, and then rename the assembly?

Would this entry point be better served if it was in the `Microsoft.ML.Ensemble` assembly?

Reply:
Good point. We've added some entry-points to legacy, but it's unclear to me whether these were intended to be legacy API entry-points only, or whether they were intended to be generally useful.

For example, we see the "model combiner" directly above this [appears in NimbusML](https://github.com/Microsoft/NimbusML/blob/e1004720ec0c252ba87f02c190c33739d9c00f20/src/python/nimbusml/internal/entrypoints/transforms_modelcombiner.py), and [same for this one I just moved you're commenting on](https://github.com/Microsoft/NimbusML/blob/e1004720ec0c252ba87f02c190c33739d9c00f20/src/python/nimbusml/internal/entrypoints/models_ovamodelcombiner.py).

So both were published (because they're entry-points after all.) But .

So it is not going to be a problem for this specific code that I just moved, at least not for this specifically, but it will definitely be a problem for these other things in this file. I've opened an issue #1565. It's not clear to me whether the usage of these entry-points in API was deliberate or a good idea, maybe someone that actually worked on NimbusML can comment more on this.